loopchain/blockchain/epoch.py

```diff
@@ -116,14 +116,13 @@ def __add_tx_to_block(self, block_builder):
                 block_builder.transactions[tx.hash] = tx
                 block_tx_size += tx.size(tx_versioner)
 
-    def makeup_block(self):
+    def makeup_block(self, complained: str or None):
```

Comment:
I thinks `complained` is bool type. right?
And it's not  correct annotation syntax.
https://stackoverflow.com/a/33945518